### PR TITLE
Add info on 2024 Dev Hackathon

### PR DESCRIPTION
**Pull request by rly**

### CODE REVIEW

**Review by oruebel — COMMENTED** [review]
Is the `nwb+dandi_logo.png` used here?

**Comment by rly** [comment]
> Is the `nwb+dandi_logo.png` used here?

No, but I figured I would copy these logo files that we use in other pages here instead of copying them into new hackathon folders as we create them. I was debating whether to update the old hackathon pages to use these links. Happy to update those pages or remove the currently unused logo files like that one.

**Comment by oruebel** [comment]
I think it's best to just keep the folders self-contained, mainly because if we change images in the shared folder we end up updating pages from previous hackathons as well.

File: HCK18_2024_Dev_Hackathon_DataJoint/README.md [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+
+## Overview
+
+The Neurodata Without Borders (NWB) team is organizing a hackathon event for developers.
+In contrast to our NWB user training workshops, this event is focused on bringing together the developers of the
+NWB data standard and software ecosystem and developers of community tools who would like to integrate their tools
+with NWB. This hackathon will enable participants to work intensively on an NWB-related project with the assistance of
+the NWB development team and others in the community. Together, we can develop and discuss ideas for solving technical
+problems that impact the broader community. Our goal is to foster collaboration and community among developers.
+
```
Comment (bendichter) [inline HCK18_2024_Dev_Hackathon_DataJoint/README.md:30]:
```suggestion

This is our first NWB hackathon event hosted by DataJoint. Working in the DataJoint building and with their team will facilitate projects at the intersection between DataJoint and NWB, such as Spyglass or building NWB export for custom DataJoint pipelines. We also welcome NWB projects that are not related to DataJoint.
```

**Review by bendichter — APPROVED** [review]

**Pull request merged**